src/DbConnectedResource.php

```diff
@@ -1,30 +1,51 @@
 <?php
 /**
  * @license   http://opensource.org/licenses/BSD-3-Clause BSD-3-Clause
- * @copyright Copyright (c) 2014 Zend Technologies USA Inc. (http://www.zend.com)
+ * @copyright Copyright (c) 2014-2016 Zend Technologies USA Inc. (http://www.zend.com)
  */
 
 namespace ZF\Apigility;
 
 use Zend\Db\TableGateway\TableGatewayInterface as TableGateway;
 use Zend\Paginator\Adapter\DbTableGateway as TableGatewayPaginator;
+use ZF\ApiProblem\Exception\DomainException;
 use ZF\Rest\AbstractResourceListener;
 
 class DbConnectedResource extends AbstractResourceListener
 {
+    /**
+     * @var string Name of collection class
+     */
     protected $collectionClass;
 
+    /**
+     * @var string Name of identifier field
+     */
     protected $identifierName;
 
+    /**
+     * @var TableGateway
```

Comment:
I think it should be: `Zend\Db\TableGateway\TableGateway` because `TableGatewayPaginator` requires this type (line 119 below)